particles/Renderer.cpp

```diff
@@ -177,7 +185,10 @@ void Renderer::init(score::gfx::RenderList& renderer)
             vec4 p = pbuf.d[index].pos;
             vec4 s = sbuf.d[index].spd;
             s += cs;
-            p += s;
+            vec4 ns = smod.value*0.01*s;
```

Comment:
maybe that computation (0.01 * s) should be done on the CPU, as it will be done only once vs for every particle 60 times per second on the GPU